lib/surface.ex

```diff
@@ -123,6 +121,18 @@ defmodule Surface do
     )
   end
 
+  defp compute_line_offset(meta) do
+    # Since `v1.11` this will create accurate line numbers for heredoc usages
+    # of the `~H` sigil. For version below `v1.11` single-line ~H will return
+    # an incorrect line number because there was no way to retrieve the information
+    # about the delimiter. See https://github.com/msaraiva/surface/issues/240
+    cond do
+      Keyword.has_key?(meta, :indentation) -> 1
+      not Version.match?(System.version(), "~> 1.11") -> 1
```

Comment:
Should we use the condition `~> 1.11` or `>= 1.11.0`? The last one will work with Elixir 2.0.